### PR TITLE
feat: restructure cmd folder

### LINKED ISSUE #7: restructure cmd folder

### PR DESCRIPTION
**Pull request by fqutishat**
closes #7

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

**Comment by alexmond** [comment]
verify-bddtest

**Comment by alexmond** [comment]
verify-bddtest

File: cmd/main.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/*
```
Comment (troyronda) [inline cmd/main.go]: This should be in a sub folder matching the binary name.

File: cmd/startcmd/startcmd_test.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/*
```
Comment (troyronda) [inline cmd/startcmd/startcmd_test.go]: This is not a name of a binary so it should not be directly under cmd

Comment (fqutishat) [reply cmd/startcmd/startcmd_test.go]: done

File: cmd/main.go (hunk 1 above)
Comment (fqutishat) [reply cmd/main.go]: done

File: cmd/blocnode/startcmd/startcmd.go [hunk 3]
```diff
@@ -0,0 +1,45 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package startcmd
```
Comment (troyronda) [inline cmd/blocnode/startcmd/startcmd.go]: Is this startcmd or ledgercmd?

Comment (fqutishat) [reply cmd/blocnode/startcmd/startcmd.go]: I think startcmd

Comment (fqutishat) [reply cmd/blocnode/startcmd/startcmd.go]: each command will have it own folder

Comment (fqutishat) [reply cmd/blocnode/startcmd/startcmd.go]: startcmd , statuscmd , versioncmd ......

Comment (fqutishat) [reply cmd/blocnode/startcmd/startcmd.go]: I change it

**Pull request merged**